seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1217,7 +1217,7 @@
         "We can also use the <code>+=</code> operator to <dfn>concatenate</dfn> a string onto the end of an existing string variable. This can be very helpful to break a long string over several lines.",
         "<strong>Note</strong><br>Watch out for spaces. Concatenation does not add spaces between concatenated strings, so you'll need to add them yourself.",
         "<hr>",
-        "Build <code>myStr</code> over several lines by concatenating these two strings:<br><code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator."
+        "Build <code>myStr</code> over several lines by concatenating these two strings: <code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator. Use the <code>+=</code> operator similar to how it is shown in the editor. Start by assigning the first string to <code>myStr</code>, then add on the second string."
```

Comment:
This really looks good 👍 

I know that this was in my suggested version as well, but now I'm starting to doubt whether this is proper grammar 😅 
> then add on the second string

Is that fine that way? I'm not a native English speaker. It doesn't feel quite right to me, but it might be OK.